diff --git a/config/database.yml b/config/database.yml
index 6d60227..8d55a2a 100644
--- a/config/database.yml
+++ b/config/database.yml
@@ -18,10 +18,10 @@ development:
   adapter: mysql
   encoding: utf8
   reconnect: false
-  database: emd_development
+  database: proficientMD_development
   pool: 5
-  username: emddevroot
-  password: 1emddevroot 
+  username: root
+  password: 123456
   socket: /var/run/mysqld/mysqld.sock
 
 # Warning: The database defined as "test" will be erased and
@@ -31,18 +31,18 @@ test:
   adapter: mysql
   encoding: utf8
   reconnect: false
-  database: emd_test
+  database: proficientMD_test
   pool: 5
-  username: emdtestroot
-  password: 1emdtestroot
+  username: root
+  password: 123456
   socket: /var/run/mysqld/mysqld.sock
 
 production:
   adapter: mysql
   encoding: utf8
   reconnect: false
-  database: emd_prod
+  database: proficientMD_production
   pool: 5
-  username: emdprodroot
-  password: 1emdprodroot
+  username: root
+  password: 123456 
   socket: /var/run/mysqld/mysqld.sock
diff --git a/db/seed.rb~ b/db/seed.rb~
deleted file mode 100644
index fc2a0b1..0000000
--- a/db/seed.rb~
+++ /dev/null
@@ -1,21 +0,0 @@
-# in db/seeds.rb
-######                        This is the db Seed File                      ##############
-# Place all Index's, application foundation configuration data, and create calls in here #
-# Also describe each  block's purpose and reference of data source                       #
-##########################################################################################
-# TEMPLATE ####################
-require 'open-uri'
-require 'active_record/fixtures'
-
-def load_seed_files!
-  seed_files = Dir["#{RAILS_ROOT}/db/seeds/all/*.rb"] + Dir["#{RAILS_ROOT}/db/seeds/#{RAILS_ENV}/*.rb"]
-  seed_files.each do |file|
-  puts "Loading seed file: #{file}"
-  load(file)
-  end
-end
-# TEMPLATE ####################
-
-
-Condition.create(:name => 'My article title', :google_url => 'Lorem ipsum dolor sit amet, consectetur adipisicing elit')
-
diff --git a/db/seed/all/01_load_users.rb b/db/seed/all/01_load_users.rb
deleted file mode 100644
index f3cb04d..0000000
--- a/db/seed/all/01_load_users.rb
+++ /dev/null
@@ -1,8 +0,0 @@
-# in db/seed/all/01_load_users.rb
-######                        This is the db Seed File                      ##############
-# Place all Index's, application foundation configuration data, and create calls in here #
-# Also describe each  block's purpose and reference of data source                       #
-##########################################################################################
-
-u = User(admin)
-u.password(admin)
diff --git a/db/seed/all/01_load_users.rb~ b/db/seed/all/01_load_users.rb~
deleted file mode 100644
index 7495a0a..0000000
--- a/db/seed/all/01_load_users.rb~
+++ /dev/null
@@ -1,8 +0,0 @@
-# in db/seed/all/users.rb
-######                        This is the db Seed File                      ##############
-# Place all Index's, application foundation configuration data, and create calls in here #
-# Also describe each  block's purpose and reference of data source                       #
-##########################################################################################
-
-u = User(admin)
-u.password(admin)
diff --git a/db/seed/development/users.rb b/db/seed/development/users.rb
deleted file mode 100644
index caefa54..0000000
--- a/db/seed/development/users.rb
+++ /dev/null
@@ -1,6 +0,0 @@
-# in db/seeds/development/users.rb
-######                        This is the db Seed File                      ##############
-# Place all Index's, application foundation configuration data, and create calls in here #
-# Also describe each  block's purpose and reference of data source                       #
-##########################################################################################
-
diff --git a/db/seed/development/users.rb~ b/db/seed/development/users.rb~
deleted file mode 100644
index 8b13789..0000000
--- a/db/seed/development/users.rb~
+++ /dev/null
@@ -1 +0,0 @@
-
diff --git a/db/seeds.rb b/db/seeds.rb
deleted file mode 100644
index fc2a0b1..0000000
--- a/db/seeds.rb
+++ /dev/null
@@ -1,21 +0,0 @@
-# in db/seeds.rb
-######                        This is the db Seed File                      ##############
-# Place all Index's, application foundation configuration data, and create calls in here #
-# Also describe each  block's purpose and reference of data source                       #
-##########################################################################################
-# TEMPLATE ####################
-require 'open-uri'
-require 'active_record/fixtures'
-
-def load_seed_files!
-  seed_files = Dir["#{RAILS_ROOT}/db/seeds/all/*.rb"] + Dir["#{RAILS_ROOT}/db/seeds/#{RAILS_ENV}/*.rb"]
-  seed_files.each do |file|
-  puts "Loading seed file: #{file}"
-  load(file)
-  end
-end
-# TEMPLATE ####################
-
-
-Condition.create(:name => 'My article title', :google_url => 'Lorem ipsum dolor sit amet, consectetur adipisicing elit')
-
